ci-operator/step-registry/openshift/upgrade/vsphere/migration-optin/openshift-upgrade-vsphere-migration-optin-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-upgrade-vsphere-migration-optin
+  steps:
+    pre:
+    - chain: ipi-vsphere-pre-stableinitial
+    - ref: storage-data-create
+    test:
+    - ref: openshift-e2e-test
```

Comment:
We already have a CI job that does e2e test after upgrade, IMO it will just slow things down here.

Reply:
Nevermind, this does not run e2e tests, it just runs upgrade with few synthetic tests.